build.sbt

```diff
@@ -41,7 +41,7 @@ lazy val runtime = Project(id = akkaGrpcRuntimeName, base = file("runtime"))
   .settings(
     // We don't actually promise binary compatibility before 1.0.0, but want to
     // introduce the tooling
-    mimaPreviousArtifacts := Set(organization.value %% "akka-grpc-runtime" % "0.7.3"),
+    mimaPreviousArtifacts := Set(organization.value %% "akka-grpc-runtime" % "0.8.0"),
```

Comment:
Shouldn't we use `previousStableVersion` here?
```
      mimaPreviousArtifacts := Set(
          organization.value %% name.value % previousStableVersion.value
            .getOrElse(throw new Error("Unable to determine previous version"))
        ),
```

Maybe always excluding internal packages would be attractive?
```
mimaBinaryIssueFilters += ProblemFilters.exclude[Problem]("akka.grpc.internal.*")
```

Reply:
Updated to use `previousStableVersion`, wasn't aware of it.

I don't think we want to exclude internal packages, as they might still have `InternalStableApi`